### PR TITLE
14.0 - [IMP] fieldservice_equipment_stock

### PR DESCRIPTION
**Pull request by marcos-mendez**
This feature allow only real PRODUCT EQUIPMENT to be indexed in the Maintenance Equipment form FSM.EQUIPMENT VIEW

### CODE REVIEW

**Comment by max3903** [comment]
@marcos-mendez `fieldservice_equipment_stock` does not depend on `maintenance` and should not depend on it.

**Comment by marcos-mendez** [comment]
@max3903 it is to allow only equipments as it has nothing to DO on Service product, Service products and consumable products shouldnt be shown in this views. 

You can test it here https://github.com/kimenz/field-service/pull/1

**Comment by max3903** [comment]
@marcos-mendez It is working on https://github.com/kimenz/field-service/pull/1 because of `('type', '=', 'product')`.

Your change is using the field `maintenance_ok`. This field does not exist if `maintenance` is not installed.

**Comment by marcos-mendez** [comment]
Yes @max3903 thats right because we were having the option to put consumable toilet paper in te fieldsetrvice equipment it was ultrageous. My boss called me and gave me a scolding. I told him i can't be responsible by the user in a Homer Simpson Style. Sorry i don't know the other stuff, tests and pre-commit trying to make my best here

**Comment by max3903** [comment]
@marcos-mendez Correction: This field does not exist even if `maintenance` is installed. Which module is adding this field?

**Comment by marcos-mendez** [comment]
In portugues this field is show as is FSM equipment

File: fieldservice_equipment_stock/views/fsm_equipment.xml [hunk 1]
```diff
@@ -8,7 +8,10 @@
         <field name="arch" type="xml">
             <group id="secondary" position="inside">
                 <group string="Inventory" groups="stock.group_stock_user">
-                    <field name="product_id" />
+                    <field 
+                        name="product_id"
+                        domain="[('maintenance_ok', '=', true)]"
```
Comment (max3903) [inline fieldservice_equipment_stock/views/fsm_equipment.xml:13]:
```suggestion
                        domain="[('type', '=', "product")]"
```

**Comment by marcos-mendez** [comment]
maintenance_ok is not the field is product is fsm.equipment here. Sorry was a mess my PR

**Comment by max3903** [comment]
@marcos-mendez Check my comments on the code. I believe that is what you are looking for.

**Comment by marcos-mendez** [comment]
Really Sorry max i dont know that much git. I'll try to make my best

**Comment by max3903** [comment]
@marcos-mendez Don't be sorry. Take your time to read the Git documentation...

File: fieldservice_equipment_stock/views/fsm_equipment.xml [hunk 2]
```diff
@@ -8,7 +8,10 @@
...
+                     />
```
Comment (max3903) [inline fieldservice_equipment_stock/views/fsm_equipment.xml]:
```suggestion
                    <field name="product_id" domain="[('type', '=', 'product')]" />
```

**Comment by max3903** [comment]
@marcos-mendez Click on "Commit suggestion", that should fix the errors...

**Comment by max3903** [comment]
@marcos-mendez Couple steps that should help you:

```
pip install pre-commit
pre-commit install
pre-commit run -a
```

**Comment by marcos-mendez** [comment]
I had never been able to install pre-commit with l10n-brazil installations because of this. 
image

**Comment by max3903** [comment]
@marcos-mendez Try with:

```
pip install --user pre-commit
```
or install `pipx` and use it to install pre-commit:

```
pipx install pre-commit
```

**Comment by marcos-mendez** [comment]
Ok it worked. But now 
image

**Comment by marcos-mendez** [comment]
Maybe i shloud close this and keep only the #1016

**Comment by marcos-mendez** [comment]
I'll open it later whe i figure it out how to make a commit only to this PR

**Pull request closed without merging**